Template/Components/Frame.html

```diff
@@ -26,7 +28,9 @@
                 <div>
                     <p>[* Εθελοντισμός */* Volunteer *]</p>
                     <div>
-                    <a href="[*localization root*][* Προσωπικό.html */* Staff.html *]">[* Προσωπικό */* Staff *]</a>
+                    <a href="[*localization root*][* Προσωπικό.html */* Short-term Staff.html *]">[* Προσωπικό */* Short-term Staff *]</a>
+                    <a href="[*localization root*][* Προσωπικό.html */* Long-term Staff.html *]">[* Προσωπικό */* Long-term Staff *]</a>
```

Comment:
Same capitalization:

```suggestion
                    <a href="[*localization root*][* Προσωπικό.html */* Long-term Staff.html *]">[* Προσωπικό */* Long‐Term Staff *]</a>
```

Also, *Προσωπικό* is being reused for two different files, so one is overwriting the other, and both links point to the same thing in Greek.